WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEmailFragment.java

```diff
@@ -125,6 +125,13 @@ protected EditText getEditTextToFocusOnStart() {
     }
 
     @Override
+    protected void onHelp() {
+        if (mLoginListener != null) {
+            mLoginListener.helpEmailScreen(EditTextUtils.getText(mEmailInput.getEditText()));
```

Comment:
Actually no, the use of the "unclean" email input is deliberate so Support will know exactly what the user inputted. I added a comment about that to make it a bit more explicit (a53ceba).